biodatacatalyst-ui/src/main/webapp/psamaui/login/fence_login.hbs

```diff
@@ -9,14 +9,14 @@
 
         <div class="center">
           <span class="fence-login-wrapper">
-            <a href="{{ fenceURL }}&idp=fence" class="fence-login-btn">
+            <a href="{{ fenceURL }}&amp;idp=fence" class="fence-login-btn">
```

Comment:
why are these being escaped?

Reply:
I accidentally checked this change in, I noticed firefox was logging these unescaped characters as errors in the console. It shouldn't matter either way, I'm fine reverting or leaving this in.